bin/server/cmd/component_omni.go

```diff
@@ -53,6 +51,20 @@ func createMetadataDB(activeProfile *Profile, logger *zap.Logger) (*metadataDB,
 	return mgr, nil
 }
 
+func (m *metadataDB) close() error {
```

Comment:
nit: put this after connect() to be consistent with the slim one and follow the logical order.